src/main/java/shukaro/warptheory/items/WarpItems.java

```diff
@@ -15,14 +16,20 @@ public static void initItems() {
         itemCleanserMinor = new ItemCleanserMinor();
         itemCleanser = new ItemCleanser();
         itemAmulet = new ItemAmulet();
-        itemWarpWardAmulet = new ItemWarpWardAmulet();
+
         itemSomething = new ItemSomething();
         itemPaper = new ItemPaper();
         GameRegistry.registerItem(itemCleanserMinor, itemCleanserMinor.getUnlocalizedName());
         GameRegistry.registerItem(itemCleanser, itemCleanser.getUnlocalizedName());
         GameRegistry.registerItem(itemAmulet, itemAmulet.getUnlocalizedName());
-        GameRegistry.registerItem(itemWarpWardAmulet, itemWarpWardAmulet.getUnlocalizedName());
+
         GameRegistry.registerItem(itemSomething, itemSomething.getUnlocalizedName());
         GameRegistry.registerItem(itemPaper, itemPaper.getUnlocalizedName());
+        if (Loader.isModLoaded("dreamcraft") == true) { // the item will not be loaded without nhcoremod, however it is
+                                                        // available to be functional for users outside GTNH being able
+                                                        // to add their own recipe for it.
```

Comment:
```suggestion
        if (Loader.isModLoaded("dreamcraft")) { // the item will not be loaded without nhcoremod, however it is
                                                // available to be functional for users outside GTNH being able
                                                // to add their own recipe for it.
```

Additionally, this doesn't do what the comment says, this prevents the item from existing at all without coremod